app/javascript/components/App.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { HashRouter, Routes, Route } from "react-router-dom";
+import NavBar from "./NabBar";
+import Doctors from "./Doctors";
+import AddDoctor from "./AddDoctor";
+import DeleteDoctor from "./DeleteDoctor";
+import ReserveForm from "./ReservseForm";
+import MyReservation from "./MyReservation";
+import DoctorDetails from "./DoctorDetails";
+import Header from "./Header";
+
+const App = () => {
```

Comment:
- [ ]  Please, kindly refactor this functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.


That way, your code will meet the latest standards. 💪